plot_types/A_basic/a_plot.py

```diff
@@ -0,0 +1,25 @@
+"""
+======================
+plot([X], Y, [fmt]...)
+======================
+"""
```

Comment:
I'd leave out the parameters in the title. They are a bit cluttered.

We should however add a bit of info inside the example, e.g.

- What this plot is. In the simplest case, this is just a repetition of the title line of the function documentation. But one might also give a little more context.
- Prominent link to the docs
- Optional: You could add the pseudo-signature from the title if you think that is helpful.

```suggestion
"""
======
plot()
======
"""
.. code-block:: none

   plot([X], Y, [fmt]...)

Plot y versus x as lines and/or markers.

Documentation: `matplotlib.axes.Axes.plot()` / `matplotlib.pyplot.plot()`
```

Reply:
This was basically copying the cheatsheet, which I think was pretty good.  If you would like to simplify it later, and add more info to the main text, I think we could do as follow-ups.